src/commands/diagnostics/metrics.ts

```diff
@@ -0,0 +1,67 @@
+import { runCommand } from '../../util/command';
+
+const outputFormat = {
+	table: [
+		{
+			header: 'Metric',
+			property: 'name'
+		},
+		{
+			header: 'Value',
+			property: 'value'
+		},
+		{
+			header: 'Type',
+			property: 'type'
+		}
+	],
+	json: [
+	]
+};
+
+export = {
+	command: 'metrics',
```

Comment:
For consistency, I'd suggest a command structure such as `skysync-cli diagnostic-metrics list`

Reply:
I followed a similar pattern to the Connection Platforms since metrics is similar in relationship to diagnostics.

Reply:
The full command is: 'skysync-cli diagnostics metrics'. There are other diagnostic endpoints that will be commands under the diagnostics namespace/container.

Reply:
Cool, that makes sense